CONTRIBUTING.md

```diff
@@ -226,4 +229,4 @@ git commit -s -S -m "Add a meaningful commit message"
 ```
 git push origin issue-number/contribution-description
 ```
-6. Open a pull request from your forked repository branch to the main branch of the 'galasa repository', and explain your changes.
+6. Open a pull request from your forked repository branch to the main branch of the 'galasa repository', and explain your changes. Refer to any stories which are relevent and 
```

Comment:
It looks like this sentence cuts off early?

Reply:
fixed